crates/solvers/src/tests/baseline/bal_liquidity.rs

```diff
@@ -53,11 +53,19 @@ async fn weighted() {
                     "sellToken": "0x6810e776880c02933d47db1b9fc05908e5386b96",
                     "buyToken": "0xdef1ca1fb7fbcdc777520aa7f396b4e015f497ab",
                     "sellAmount": "1000000000000000000",
+                    "fullSellAmount": "1000000000000000000",
                     "buyAmount": "1",
+                    "fullBuyAmount": "1",
+                    "feePolicies": [],
+                    "validTo": 0,
                     "kind": "sell",
+                    "owner": "0x5b1e2c2762667331bc91648052f646d1b0d35984",
                     "partiallyFillable": false,
+                    "preInteractions": [],
+                    "postInteractions": [],
                     "class": "market",
-                    "feePolicies": [],
+                    "signingScheme": "presign",
+                    "signature": "0x",
```

Comment:
Quasilabs stumbled over this previously. Technically there is no additional signature associated with the order but in the SC presign order will use the owner address as the signature. Might be nice to just pre-fill this accordingly.

Reply:
Yes, if we do that we should also update the orderbook format. We can leave the old format for this PR and discuss the change further but overall I like the concept that whatever is in the `signature` field is what you have to encode on-chain as well (even if it's technically duplicate data like for pre-sign).

Reply:
I completely understand. I created an issue in order to not forget and discuss it further: https://github.com/cowprotocol/services/issues/2626